Gemfile

```diff
@@ -19,7 +19,7 @@ gem 'omniauth-rails_csrf_protection'
 gem 'bootstrap',                  '>= 4.4.1'
 gem 'uglifier',                   '~> 3.0'
 gem 'friendly_id',                '~> 5.1'
-gem 'cancancan',                  '~> 1.15'
+gem 'cancancan'
```

Comment:
@KevinMulhern Can we pin it to a specific version?

Btw, weren't there any migration required as we are bumping from v1 to v3?

Reply:
Sorry I should have told you I'm trying to remove the gem restrictions for anything that we don't need to lock down. ie rails and bootstrap. Then we can use something like dependabot to regularly keep the dependencies up to date.

Nope, no migrations required. We are only using cancancan in one place in the app and it didn't affect that.

Reply:
Gotcha.